scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -315,8 +316,8 @@ class ScalaTestFramework extends SbtFramework {
    * Returns an <code>org.scalatools.testing.Runner</code> that will load test classes via the passed <code>testLoader</code>
    * and direct output from running the tests to the passed array of <code>Logger</code>s.
    */
-  def testRunner(testLoader: ClassLoader, loggers: Array[Logger]) = {
-    new ScalaTestRunner(testLoader, loggers)
+  def testRunner(testLoader: ClassLoader | Null, loggers: Array[Logger | Null] | Null) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.